gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -250,7 +259,11 @@ private void handleDagAction(DagActionStore.DagAction dagAction, boolean isStart
         throw new RuntimeException(String.format("Received LAUNCH dagAction while not in multi-active scheduler "
             + "mode for flowAction: %s", dagAction));
       }
-      submitFlowToDagManagerHelper(dagAction, isStartup);
+      if (dagProcessingEngineEnabled) {
+        this.dagTaskStream.addDagAction(dagAction);
+      } else {
+        submitFlowToDagManagerHelper(dagAction, isStartup);
+      }
```

Comment:
in this PR we should still have it in "test" mode so if dagProcessingEngine is labeled we add some logging around the code paths it follows and we can trace it but not necessarily change the path yet. What do you think?